packages/platform-sdk-generator/src/ir/Namespace.ts

```diff
@@ -164,4 +164,6 @@ export interface StaticOperation {
   requestBody?: RequestBody;
   response: Response;
   auth: Auth;
+  auditCategory: string;
```

Comment:
What is audit category for?

Reply:
no idea, i just saw it in the raw data and was being consistent.